src/main/webapp/app/forms/form-status-bar/form-status-bar.component.html

```diff
@@ -1,20 +1,19 @@
-<div class="form-status-bar-container pt-2 pb-1">
-    <hr class="form-status-line" />
-    <div class="d-flex justify-content-between">
-        @for (formStatusSection of formStatusSections; track formStatusSection.title) {
-            <div class="d-flex flex-column align-items-center">
-                <div
-                    class="d-flex align-items-center justify-content-center border rounded-circle border-secondary form-status-circle"
-                    (click)="scrollToHeadline(formStatusSection.title)"
-                >
-                    <jhi-checklist-check
-                        [checkAttribute]="formStatusSection.valid"
-                        [iconColor]="formStatusSection.empty && formStatusSection.valid ? 'var(--bs-gray)' : undefined"
-                        class="fa-sm"
-                    />
+@if (formStatusSections) {
```

Comment:
Only added the if check, if errors in components occur that use this component and formStatusSection is undefined the status bar breaks otherwise